app/Notifications/NewRequestFill.php

```diff
@@ -42,6 +43,24 @@ public function via(object $notifiable): array
         return ['database'];
     }
 
+    /**
+     * Determine if the notification should be sent.
+     */
+    public function shouldSend(User $notifiable): bool
+    {
+        if ($notifiable->notification?->block_notifications == 1) {
+            return false;
```

Comment:
Should probably add the check to not notify yourself filling your own request, similar to how it's done before this pr.

Reply:
Added self check.